invenio_oauthclient/contrib/cern.py

```diff
@@ -209,12 +209,19 @@ def get_resource(remote):
 def account_info(remote, resp):
     """Retrieve remote account information used to find local user."""
     res = get_resource(remote)
+    session.pop('cern_resource')
 
     email = res['EmailAddress'][0]
     external_id = res['PersonID'][0]
     nice = res['CommonName'][0]
     name = res['DisplayName'][0]
 
+    groups = fetch_groups(res['Group'])
```

Comment:
Ok, I understand what the problem is now. The solution is not to move the code to ``account_info`` because it opens up the code for attacks.

I'm not sure what would be best way to solve it, but it could be to do the groups fetching in a [Flask-Login user_logged_in signal](https://flask-login.readthedocs.io/en/latest/#signals)

CDS (@egabancho @ludmilamarian) I think must have the same problem or already have it solved.